src/components/atoms/ScrollToTop/index.jsx

```diff
@@ -0,0 +1,39 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { withRouter } from "react-router-dom";
+//#endregion
+
+//#region > Components
+/**
+ * @class ScrollToTop Component
+ * @description When reloading or switching a page, ReactJS now starts at the
+ *              top of the page, and no longer on the last position.
+ */
+class ScrollToTop extends React.Component {
+  componentDidUpdate(prevProps) {
+    const { location } = this.props;
+
+    if (location !== prevProps.location) {
+      // Scroll to top of viewport
+      window.scrollTo(0, 0);
+    }
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+//#endregion
+
+//#region > Exports
+//> Default Class
+export default withRouter(ScrollToTop);
```

Comment:
You are switching between coding guide lines in the files, either choice to write this comment for every export or remove it from every export.

Reply:
Btw you are not exporting a default class here. It don't know exactly what withRouter() returns but it is not a class.

Reply:
@schettn I copied this from `snek-at/front`.